From 69a75f2d8b50cd69224a04a5a3a22ee747f4da02 Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Thu, 2 Jul 2020 08:04:19 -0500
Subject: [PATCH 45/49] arm64: dts: beacon-imx8mm: Remove GPIO pinmuxing for
 I2C4

The i2c4 controller can directly drive the i2c4 pins without
the gpio assignments.

This patch removes them.

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 .../boot/dts/freescale/beacon-imx8mm-baseboard.dtsi    | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/beacon-imx8mm-baseboard.dtsi b/arch/arm64/boot/dts/freescale/beacon-imx8mm-baseboard.dtsi
index 7a8aa3f553a1..6dac7281ebcf 100644
--- a/arch/arm64/boot/dts/freescale/beacon-imx8mm-baseboard.dtsi
+++ b/arch/arm64/boot/dts/freescale/beacon-imx8mm-baseboard.dtsi
@@ -225,13 +225,6 @@
 		>;
 	};
 
-	pinctrl_i2c4_gpio: i2c4grp-gpio {
-		fsl,pins = <
-			MX8MM_IOMUXC_I2C4_SCL_GPIO5_IO20        	0x1c3
-			MX8MM_IOMUXC_I2C4_SDA_GPIO5_IO21        	0x1c3
-		>;
-	};
-
 	pinctrl_pcal6414: pcal6414-gpio {
 		fsl,pins = <
 			MX8MM_IOMUXC_SAI2_MCLK_GPIO4_IO27        	0x19
@@ -417,9 +410,6 @@
 	clock-frequency = <384000>;
 	pinctrl-names = "default", "gpio";
 	pinctrl-0 = <&pinctrl_i2c4>;
-	pinctrl-1 = <&pinctrl_i2c4_gpio>;
-	scl-gpios = <&gpio5 20 GPIO_ACTIVE_HIGH>;
-	sda-gpios = <&gpio5 21 GPIO_ACTIVE_HIGH>;
 	status = "okay";
 
 	pca6416_0: gpio@20 {
-- 
2.17.1

